.azure/lint-linux.yml

```diff
@@ -35,7 +35,7 @@ jobs:
           set -e
           source test-job/bin/activate
           echo "Running black, any errors reported can be fixed with 'tox -eblack'"
-          black --check qiskit test tools examples setup.py
+          black --check qiskit test tools examples setup.py qiskit_pkg/setup.py
```

Comment:
This file needs re-syncing with the `tox.ini` rules for lint.

Reply:
Done in: https://github.com/Qiskit/qiskit-terra/pull/10530/commits/db905383f4ab1bab28a795da3f20d7b0162917aa